guide-us-english.tex

```diff
@@ -75,7 +75,7 @@ \subsection{Definition of Scrum@Scale}
 
 The Scrum Guide is the minimal feature set that allows inspection and adaptability via radical transparency to drive innovation, customer satisfaction, performance, and team happiness.
 
-\textbf{Scrum@Scale}: A framework within which networks of Scrum Teams operating consistently with the Scrum Guide can address complex adaptive problems, while creatively delivering products of the highest possible value.
```

Comment:
I do like mentioning the Scrum Guide here, since it clarifies the relationship between the guides.